README.md

```diff
@@ -748,6 +748,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Guava](https://github.com/google/guava) - Collections, caching, primitives support, concurrency libraries, common annotations, string processing, I/O, and so forth.
 * [JADE](http://jade.tilab.com/) - Framework and environment for building and to debugging multi-agent systems.
 * [JavaVerbalExpressions](https://github.com/VerbalExpressions/JavaVerbalExpressions) - A library that helps to construct difficult regular expressions.
+* [minio-java](https://github.com/minio/minio-java) - Minio Java Library for Amazon S3 compatible cloud storage.
```

Comment:
Would make more sense to add the description from the repository:

`Provides simple APIs to access any Amazon S3 compatible object storage server.`